### PR TITLE
Updates in anticipation of cnab-core-1.1.0

### PR DESCRIPTION
**Pull request by vdice**
Docs and other updates in anticipation of the forthcoming/proposed cnab-core-1.1.0 release.

The main spec change prompting this minor release is the following clarification: https://github.com/cnabio/cnab-spec/issues/287

Ref https://github.com/cnabio/cnab-spec/issues/387

### CODE REVIEW

File: 000-index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ weight: 001
 
 # CNAB Specifications
 
-1. [Cloud Native Application Bundle Core 1.0.0 (CNAB1)](100-CNAB.md)
+1. [Cloud Native Application Bundle Core 1.1.0 (CNAB1)](100-CNAB.md)
```
Comment (vdice) [inline 000-index.md:8]: Q: Does `CNAB1` need to change?

File: 101-bundle-json.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The `bundle.json` file is a representation of bundle metadata. It MUST be repres
 
 A `bundle.json` is broken down into the following categories of information:
 
-- The schema version of the bundle, as a string with a `v` prefix. This schema is to be referenced as `v1` or `v1.0.0`
+- The schema version of the bundle, as a string with a `v` prefix. This schema is to be referenced as `v1` or `v1.1.0`
```
Comment (vdice) [inline 101-bundle-json.md:16]: I wasn't sure if all of these `v1.0.0` references should be updated to `v1.1.0` or not?

File: README.md [hunk 3]
```diff
@@ -4,15 +4,16 @@
 
 Cloud Native Application Bundles (CNAB) are a package format specification that describes a technology for bundling, installing, and managing distributed applications, that are by design, cloud agnostic.
 
-## CNAB Core 1.0 (Final)
+## CNAB Core 1.1.0 (Final)
 
-The CNAB Working Group with the joint approval of the Executive Directors has approved the CNAB Core 1.0 specification for publication. CNAB Core 1.0 is complete.
+The CNAB Working Group with the joint approval of the Executive Directors has approved the CNAB Core 1.1.0 specification for publication. CNAB Core 1.1.0 is complete.
```
Comment (vdice) [inline README.md:9]: Should I have added an additional section rather than amending this one?

File: README.md [hunk 4]
```diff
@@ -4,15 +4,16 @@
...
 
-For more information on the approval process, see [the process documentation](901-process.md). Further changes to CNAB Core will be considered for CNAB Core 1.1.
+For more information on the approval process, see [the process documentation](901-process.md). Further changes to CNAB Core will be considered for CNAB Core 1.2.
 
 ### Branch/Tag Structure
 
 - [master](https://github.com/cnabio/cnab-spec) is the current working draft of all specs
+- [cnab-core-1.1.0](https://github.com/cnabio/cnab-spec/tree/cnab-core-1.1.0) is the CNAB Core 1.1.0 final draft
```
Comment (vdice) [inline README.md:16]: This tag is yet to be created; the intention is to cut the tag after this PR is approved, merged and included in the forthcoming `cnab-core-1.1.0-ga` branch.  Ref ordering in https://github.com/cnabio/cnab-spec/issues/387

File: 000-index.md (hunk 1 above)
Comment (chris-crone) [reply 000-index.md:8]: I would prefer if we have CNAB<major> mean the latest CNAB release of the <major> version (i.e.: In this case, CNAB1 means CNAB 1.1.0) as I think that usually this is what people mean when they say version <major>. If one wants to be more specific, they can do so by specifying the minor and patch versions as well.

If we choose to do this, we may need to clarify our [process versioning](https://github.com/cnabio/cnab-spec/blob/master/901-process.md#versioning).

File: 101-bundle-json.md (hunk 2 above)
Comment (chris-crone) [reply 101-bundle-json.md:16]: While the schema has not changed, it is versioned with the core of the spec so I think it should match that (i.e.: 1.1.0).

File: README.md (hunk 3 above)
Comment (chris-crone) [reply README.md:9]: I think amending here makes sense. We don't want to pollute the README over time with a release history.

**Review by chris-crone — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (technosophos) [reply README.md:9]: Agree

**Review by technosophos — APPROVED** [review]
Sorry that it took me so long to review this, but it looks good as-is.

**Comment by vdice** [comment]
@carolynvs @radu-matei @jlegrone Can we get your approval on this PR/CNAB Core 1.1.0 release?

The changelog prompting this minor release is:

- docs(101-bundle-json.md): add clarification around contentDigest value https://github.com/cnabio/cnab-spec/pull/384

**Pull request merged**